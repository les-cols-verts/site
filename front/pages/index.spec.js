import index from '@/pages/index.vue';

jest.mock('@/components/modules/Home/Map/Map', () => () => ({
  Map: '<div data-test="map"/>',
}));

describe('index', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(index);

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
