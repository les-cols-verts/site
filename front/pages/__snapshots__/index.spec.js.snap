// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should match snapshot 1`] = `
<div class="container">
  <app-header-stub></app-header-stub>
  <home-stub></home-stub>
</div>
`;
